.github/workflows/ci-pr.yml

```diff
@@ -66,6 +68,7 @@ jobs:
         raco test -l tests/db/all-tests
         raco test -c tests/stxparse
         raco test -c tests/syntax
+    # ???
```

Comment:
Question in general: if the file/directory is not that large, is it OK to `upload-artifact` without making a Tarball first? 

(I know in this particular case we definitely should do it, since the directory is pretty large)

Reply:
We could although in general, I prefer to create a tarball. I always slightly worried about how folders/files/symlinks will be handled between OSes.